### ISSUE TITLE
Re-enable Erlang benchmark

### ISSUE THREAD

**Issue by LesnyRumcajs** [LesnyRumcajs/grpc_bench#147]
Erlang benchmark stopped working (on my local machine all requests failed according to ghz report). Would be nice to identify the issue and resolve it. I temporarily disabled it to unblock CI.

**Comment by LesnyRumcajs** [comment]
Perhaps @fenollp , the local Erlang champion, you could take a look in spare time? :)

**Comment by fenollp** [comment]
Ah indeed there is an issue with unary calls made from a Go client. I am tracking [this here](https://github.com/tsloughter/grpcbox/issues/51) and will dig further once I find some time.

**Comment by LesnyRumcajs** [comment]
@fenollp thanks!

**Comment by tsloughter** [comment]
Yo, I'm the author of grpcbox and hope to help. I've not been doing much of anything recently though because I've been on paternity leave :), but want to get this figured out. I see I've already been ignoring an issue on the repo itself :(

**Comment by fenollp** [comment]
Hi Tristan! Congrats :) and thanks for responding. 

As you note there's this issue of the gPRC stream being unexpectedly reset during unary calls from a gRPC-go client. 

If you can find some time to look into this that'd be very helpful :)

**Comment by Trisfald** [comment]
Closing this since the erlang benchmark works now.

**Comment by fenollp** [comment]
I should have linked the PR to this issue.
FYI grpcbox (actually, chatterbox) still requires some patching. [See the PR's dependency override.](https://github.com/LesnyRumcajs/grpc_bench/pull/174/files#diff-2022b513427d1165f5925bda174bb7af40ab433e23eb0f5a230c93e7695a206fR25)

**Comment by tsloughter** [comment]
Weird, I see I commented back when I was on my first paternity leave and now I've run into this again via a tweet during my second leave :). 

Anyway, I saw our numbers are the worst, hehe. Recently had some major changes to the HTTP/2 client and server merged so hopefully that is going to change.

I haven't published the new versions to hex yet but opened up a PR to bump the benchmark https://github.com/LesnyRumcajs/grpc_bench/pull/363
